test/SILOptimizer/templvalueopt_ossa.sil

```diff
@@ -0,0 +1,251 @@
+// RUN: %target-sil-opt -enable-sil-verify-all %s -temp-lvalue-opt | %FileCheck %s
+
+import Swift
+import Builtin
+
+// CHECK-LABEL: sil [ossa] @test_enum_with_initialize
+// CHECK:      bb0(%0 : $*Optional<Any>, %1 : $*Any):
```

Comment:
nit : Add : after `// CHECK-LABEL: sil [ossa] @test_enum_with_initialize`
Same in all functions below

Reply:
Thanks! You are right! I should have fixed! I'll fix after the tests pass and then do a smoke test